tools/monorepo-utils/src/code-freeze/commands/changelog/lib/index.ts

```diff
@@ -5,6 +5,7 @@ import simpleGit from 'simple-git';
 import { execSync } from 'child_process';
 import { readFile, writeFile } from 'fs/promises';
 import path from 'path';
+import { readFileSync } from 'fs';
```

Comment:
Generic feedback: the workflow `Install prerequisites` step shouldn't execute `pnpm build` - we typically push update dbuild artifacts (under `tools/monorepo-utils/dist`) along with the changes, like we introduced in this PR. This helps to simplify setup steps and speed up CI.